acra/configuring-maintaining/general-configuration/necessary_prep.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Necessary preparations
+bookCollapseSection: true
+weight: 1
+---
+
+# Necessary preparations (keys, ports, etc.)
+
+## Master key
+
+Most Acra services require master key in order to work and perform cryptographic operations.
+It is used, for example, to encrypt other keys in their keyrings.
+These keys are then used in more specific tasks
+(encrypt data in database, hash data for searchable encryption).
```

Comment:
I'd say these are completely separate topics:

- generating and managing your master key
- managing Acra keystore (which is related)
- what ports you should open for Acra to function

And they should be covered on separate pages, I believe.